### ISSUE TITLE
Commits timestamp in local time zone

### ISSUE THREAD

**Issue by Flavio78** [phil294/git-log--graph#54]
How coul I see my commits timestamp in my local time zone?

Now I see this:
<img width="194" alt="image" src="https://github.com/phil294/git-log--graph/assets/53538168/6e865a0c-d499-4e14-b6d9-c03d90289462">

But I would like to see in my time zone as got from `git log --date=local --graph`

2023-08-28 20:28:24
2023-08-28 20:21:07
2023-08-28 19:34:59
... and so on.

I edited the default configuration adding `--date=local` but nothing changed.

Thanks.

**Comment by phil294** [comment]
right, it *should* be respecting `--date=local` but right now we only have hard coded date formatting I think. Definitely needs to be fixed

**Comment by Flavio78** [comment]
Thanks for the answer. Best regards.

**Comment by phil294** [comment]
fixed now with v0.1.13. Default is now local timezone (seems preferable to me), and you can switch back to author timezone by changing `--date=iso-local` to `--date=iso`

**Comment by Flavio78** [comment]
Thanks @phil294 .
